src/components/workAllocation/WorkAllocation.tsx

```diff
@@ -206,7 +211,24 @@ export default function WorkAllocation() {
   return (
     <div>
       <div className="mx-auto max-w-screen-lg mt-2 my-6 border border-gray-200 bg-gray-100 p-6 rounded-md space-y-4">
-        {successMessage && <Success message={successMessage} />}
+        {successMessage && (
+          <>
+            <Success message={successMessage} />
+            {userRoleIncludes(UserRole.Enduser) && (
+              <div className={'flex flex-row text-pink-600 mt-4'}>
+                <InfoIcon className={`bg-white inline-block '}`} />
+                <a
+                  href={
+                    'https://fred.wellcomegenomecampus.org/Interact/Pages/Content/Document.aspx?id=6817&utm_source=interact&utm_medium=side_menu_category'
+                  }
+                  className="text-sp-600 hover:text-sp-700 font-semibold hover:underline text-base tracking-wide"
+                >
+                  Please complete RNAscope/IHC template for probes/antibody and fluorophore
```

Comment:
Can this be styled to look less like an error message please?

Reply:
Ok, More like an information I hope.

Reply:
Done